test/QuestBudget.t.sol

```diff
@@ -368,30 +391,21 @@ contract QuestBudgetTest is Test, TestUtils, IERC1155Receiver {
         questBudget.reclaim(data);
     }
 
-    ///////////////////////////
+    //////////////////////////////////
     // QuestBudget.createERC20Quest //
-    ///////////////////////////
+    //////////////////////////////////
     function testCreateERC20Quest() public {
-        // Define the parameters for the new quest
-        uint32 txHashChainId_ = 1;
```

Comment:
Common parameters for quest creation are moved to helper functions at the bottom of this file `setupQuestData` and `createQuest`. This is to avoid having to repeat the same quest setup on multiple tests.